### PR TITLE
Adds user group name to edly cookies

### PR DESCRIPTION
**Pull request by rehan99000**
**Description:**
This PR adds logged in user's group name to edly cookies for use in other services.

### CODE REVIEW

**Review by asadmanzoor93 — APPROVED** [review]

**Review by burhandodhy — APPROVED** [review]

File: openedx/features/edly/cookies.py [hunk 1]
```diff
@@ -78,6 +78,7 @@ def _get_edly_user_info_cookie_string(request):
                 request.user,
                 CourseCreatorRole()
             ) if getattr(request, 'user', None) else False,
+            'group_name': request.user.groups.first().name if getattr(request, 'user', None) else '',
```
Comment (asadmanzoor93) [inline openedx/features/edly/cookies.py:81]: @rehan99000 The case of multiple permission groups will be missed.. you are only adding the first group name, while a user can belong to multiple permission groups. Kindly address that.

**Review by asadmanzoor93 — CHANGES_REQUESTED** [review]
Added my initial review.

File: openedx/features/edly/cookies.py (hunk 1 above)
Comment (rehan99000) [reply openedx/features/edly/cookies.py]: added all groups

**Review by asadmanzoor93 — CHANGES_REQUESTED** [review]
Kindly fix the failing quality tests.

**Review by asadmanzoor93 — APPROVED** [review]

**Pull request merged**